vm/systemSmartContracts/validator.go

```diff
@@ -175,12 +191,22 @@ func NewValidatorSmartContract(
 		enableUnbondTokensV2Epoch:        args.EpochConfig.EnableEpochs.UnbondTokensV2EnableEpoch,
 		validatorToDelegationEnableEpoch: args.EpochConfig.EnableEpochs.ValidatorToDelegationEnableEpoch,
 		shardCoordinator:                 args.ShardCoordinator,
+		stakeLimitsEnableEpoch:           args.EpochConfig.EnableEpochs.StakeLimitsEnableEpoch,
+		nodeLimitPercentage:              args.StakingSCConfig.NodeLimitPercentage / 100.0,
```

Comment:
there is a divide by 100 here, but on line 199, there is no /100 on the stakeLimitPercentage

Reply:
there is no need as core.GetIntTrimmedPercentageOfValue resolves the 100

Reply:
deleted the divide here

Reply:
ok!